src/components/Footer/Footer.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { AiFillGithub, AiFillTwitterCircle, AiFillLinkedin } from 'react-icons/ai';
+
+import { CompanyContainer, FooterWrapper, LinkColumn, LinkItem, LinkList, LinkTitle, Slogan, SocialContainer, SocialIcons, SocialIconsContainer } from './FooterStyles';
+
+const Footer = () => {
+  return (
+    <FooterWrapper>
+      <LinkList>
+        <LinkColumn>
+          <LinkTitle>Call</LinkTitle>
+          <LinkItem href="tel:402-718-0914">402-718-0914</LinkItem>
```

Comment:
Putting your phone and email on your site like this will cause crawlers to find and sell your contact info to spammers. Recommend obfuscating from crawlers somehow or just using a contact form.